solidity/dashboard/src/sagas/web3.js

```diff
@@ -9,7 +9,7 @@ import {
 } from "../actions/messages"
 import { messageType } from "../components/Message"
 
-function createTransactionEventChannel(contract, method, args, options) {
+function createTransactionEventChannel(contract, method, args = [], options) {
```

Comment:
In general I tend towards having a default value for all arguments after the first one with default value, just so it's clear to someone looking at it what's going on. Would it make sense to give `options` a default value of `{}`?

Reply:
> Would it make sense to give options a default value of {}?

Let's make this change in a follow-up---I'd like to get the release tokens bug fix deployed.